Composer/packages/lib/ui-shared/src/components/tags/utils.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const removeLineBreaks = (value: string) => {
+  return value.replace(/(\r\n|\n|\r)/gm, '');
+};
+
+// TAKEN FROM - https://github.com/janl/mustache.js/blob/master/mustache.js#L55
+const htmlEntityMap: Record<string, string> = {
```

Comment:
This seems useless.. Can you explain what the intent is?

Reply:
makes it easier to extend and look up is o(1)